### DISCUSSION TITLE
Upgrading 1.15.3 to 1.15.4: Error:  Clearing the cache for the prod environment with debug false

### DISCUSSION THREAD

**Discussion by ralf-e** [kimai/kimai#2836] [Questions & Answers]
When updating from 1.15.3 to 1.15.4 I get this error message and Kimai can no longer be reached.

composer install --no-dev --optimize-autoloader was still running without an error message.

With "bin / console kimai: update" then the error.

Calling cache: clear manually does not work either.
Calling ... install ... again now returns the same errors.

(PHP version 7.3.30, Hosting: all-inkl.com)

How can I get Kimai to work again without losing my times?

![grafik](https://user-images.githubusercontent.com/3521615/136688992-73a25c07-b3eb-4e97-a0d6-bed3f648a39a.png)
![grafik](https://user-images.githubusercontent.com/3521615/136689005-7132309f-d454-4be0-821e-cd76f5b5172d.png)

**Comment by kevinpapst** [comment]
1. The times are in your database, no worry that you could loose them (even though: backups don't hurt!).
2. What does `git status` report?

**Comment by ralf-e** [comment]
git status:
![grafik](https://user-images.githubusercontent.com/3521615/136689965-38d1f381-81db-47b1-9e12-843e0322332c.png)

**Reply by kevinpapst** [reply]
Yeah well, that looks very wrong. 
Backup all files that you modified yourself (eg .env, local.yaml) and then execute a `git reset --hard` followed by the normal upgrade process.

**Reply by ralf-e** [reply]
Thank you very much, Kimai is now working again ...

**Comment by ralf-e** [comment]
happy too early.
Everything seems to work fine online via a browser.

Synchronizing the times from the mobile phone via APP does not work:
Also with a different mobile phone, a newly installed APP and a new user / API password created in Kimai, the same error message

Complement:
After every call of "kimai: update" (cache cleared / warmed) only exactly one time entry is synchronized, then the API error comes again in the app ...

![grafik](https://user-images.githubusercontent.com/3521615/137001858-40223c5d-3252-4c3c-afe2-e5ffc915079e.png)

**Reply by kevinpapst** [reply]
I am not the app developer, but I'd bet that you have wrong file permissions.
Which you can see in your log file at var/log/prod.log (if it is missing, it is the same).
